### PR TITLE
Fix issues with nested polymorphic for sqlalchemy 1.3 and 1.4

### PR DESCRIPTION
**Pull request by lvoliveira**
* The previous solution for the load of nested polymorphic brake on sqlalchemy 1.4. The atempted solution to use inspect only work for version 1.4 and brakes on version 1.3. These changes fix the issue for both versions.

* Also fix the problem for when the column name on the table is different of the mapper class attribute name used to map the column. The serializer needs to know the later, but the former was being returned insted.

EDEN-2586

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -30,7 +31,6 @@
         "Intended Audience :: Developers",
         "License :: OSI Approved :: MIT License",
         "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
```
Comment (nicoddemus) [inline setup.py:33]: Weird, we support 3.6 and 3.8+ but not 3.7?

Comment (lvoliveira) [reply setup.py:33]: There is not a real issue, we will drop support for python 3.6 soon, and support only 3.8+. But I can leave python 3.7 build for now. No problem.

Comment (nicoddemus) [reply setup.py:33]: Yeah better to cleanly drop 3.6 and 3.7 together when the time comes

**Review by lincrosenbach — APPROVED** [review]
🎉

**Review by brnomendes — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (lvoliveira) [reply setup.py]: done

**Review by nicoddemus — APPROVED** [review]

**Pull request merged**